app/src/components/Search/PokemonHit.js

```diff
@@ -56,150 +36,127 @@ function getRefinementName(value) {
 export const PokemonHit = connectCurrentRefinements(
   ({ items: refinements, pokemon }) => {
     const spriteUrl = useShiny(pokemon.artworkUrl, pokemon.spriteShinyUrl);
-    const color = COLORS_BY_TYPE[pokemon.types[0].name.toLowerCase()];
     const statRefinements = refinements.filter(refinement =>
       refinement.attribute.startsWith("stats.")
     );
 
     return (
-      <div
-        style={{
-          borderColor: lighten(0.2, saturate(0.25, color)),
-          background: lighten(0.4, color)
-        }}
-      >
+      <div className="dcard w-1/5 mt-24">
         <PokemonLink pokemonId={pokemon.id}>
-          <header>
-            <h3>
-              <Highlight tagName="mark" attribute="names.en" hit={pokemon} />
-            </h3>
-
-            <p>
-              <Highlight tagName="mark" attribute="names.fr" hit={pokemon} />{" "}
-              <Tag>
-                fr
-              </Tag>
-              {pokemon.names.ja && (
-                <>
-                  {" "}
-                  /{" "}
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+          <div className="trigger"></div>
+
+          <div className={`card p-6 m-2 bg-gray-900 rounded-lg`}>
+            <div className="frame">
+              <img className="-mt-32" src={spriteUrl} alt={pokemon.names.en} />
+
+              <div className="card-title">
+                <div
+                  className={`pokemon-name text-center text-4xl mb-2 text-type-${pokemon.types[0].name.toLowerCase()}`}
```

Comment:
Can we extract this code `pokemon.types[0].name.toLowerCase()` to a variable ?

Will be useful for fixing my comment below